### PR TITLE
doc: adjustments of migration guide

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 1]
```diff
@@ -3,9 +3,20 @@ project.description: Migrating to Akka 2.6.
 ---
 # Migration Guide 2.5.x to 2.6.x
 
-It is now recommended to use @apidoc[akka.util.ByteString]`.emptyByteString()` instead of
-@apidoc[akka.util.ByteString]`.empty()` when using Java because @apidoc[akka.util.ByteString]`.empty()`
-is [no longer available as a static method](https://github.com/scala/bug/issues/11509) in the artifacts built for Scala 2.13.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]: Funny to start the migration guide with this, moved down.

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 2]
```diff
@@ -74,11 +85,11 @@ After being deprecated since 2.5.0, the following have been removed in Akka 2.6.
 * `akka.stream.actor.ActorSubscriber` and `akka.stream.actor.ActorPublisher`
     - Use `GraphStage` instead.
 
-After being deprecated since 2.4.0, the following have been removed in Akka 2.6.
+After being deprecated since 2.4.0, the following have been removed in Akka 2.6.0.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]: we used a mix of version styles here. Changed to 3 digits, either 2.6.x or 2.6.0 depending on context

File: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md [hunk 3]
```diff
@@ -207,12 +226,6 @@ The configuration for Artery is different, so you might have to revisit any cust
 @ref:[reference configuration for Artery](../general/configuration-reference.md#config-akka-remote-artery) and
 @ref:[reference configuration for classic remoting](../general/configuration-reference.md#config-akka-remote).
 
-@@@ note
-
-For more details on rolling updates with this migration see the @ref:[shutdown and startup](../additional/rolling-updates.md#migrating-from-classic-remoting-to-artery) section.
-
-@@@
```
Comment (patriknw) [inline akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md]: this was pretty much just linking back here (cycle)

**Review by johanandren — APPROVED** [review]
LGTM

**Pull request merged**